src/screens/BiometricSettings/BiometricSettings.tsx

```diff
@@ -0,0 +1,95 @@
+import { isBiometricSupported } from '../../util/biometric';
+import { colors } from '../../util/colors';
+import styles from './styles';
+
+const BiometricSettings = ({ navigation }) => {
+  const [isBiometricSupport, setIsBiometricSupport] = useState<boolean>(false);
+  const [usingStatus, setUsingStatus] = useState<boolean>(false);
+  const dispatch = useAppDispatch();
+  const lastLoggedUsername = useAppSelector(selectLastLoggedUsername);
+  const biometricUseStatus = useAppSelector(selectBiometricUseStatus);
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      ...backArrow({
+        navigation: navigation,
+        color: colors.primaryDark,
+      }),
+    });
+  }, []);
+
+  useEffect(() => {
+    (async () => {
+      const isBiometricSupportResult = await isBiometricSupported();
+      setIsBiometricSupport(isBiometricSupportResult);
+      if (lastLoggedUsername) {
+        setUsingStatus(!!biometricUseStatus?.[lastLoggedUsername]);
+      }
+    })();
+  }, [biometricUseStatus, lastLoggedUsername]);
+
+  const handleUsingStatusChange = async (updatedStatus: boolean) => {
+    setUsingStatus(updatedStatus);
+    if ((isBiometricSupport && lastLoggedUsername) || 1) {
+      await LocalAuthentication.authenticateAsync({
+        promptMessage:
+          'To change Telios uses settings, you must have identity verification.',
+        cancelLabel: 'Cancel',
+        disableDeviceFallback: false,
+      }).then(async res => {
+        if (res?.success && lastLoggedUsername) {
+          const newBiometricUseStatus = cloneDeep(biometricUseStatus);
+          newBiometricUseStatus[lastLoggedUsername] = updatedStatus;
+          dispatch(updateBiometricUseStatus(newBiometricUseStatus));
+          await storeAsyncStorageBiometricUseStatus(
+            lastLoggedUsername,
+            updatedStatus,
+          );
+        }
+      });
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.settingContainer}>
+        <View>
+          <Text style={styles.settingText}>Using status</Text>
+          {!isBiometricSupport && (
+            <Text style={styles.errorText}>
+              {'Your device does not include this feature.'}
+            </Text>
+          )}
+        </View>
+        <Switch
+          value={usingStatus}
+          trackColor={{
+            false: colors.inkLighter,
+            true: colors.primaryBase,
+          }}
+          disabled={!isBiometricSupport}
+          onValueChange={handleUsingStatusChange}
+        />
+      </View>
+      <Text style={styles.description}>
+        {
+          'Allows you to login to your Telios account with biometric verification. \n\nWhen you want to log in to your account after closing the Telios application, you must log in with biometric verification again. \n\n *Biometric verification allows you to log into your account using whatever security method is defined on your phone.'
```

Comment:
Using "/n" is not a flexible solution, in terms of "Flex-box". On small screen devices it won't work as expected. Please use text alignment, if it's needed use multiple <Text> tags

Reply:
Description of biometric usage is too long and unclear. I suggest "You will be able to login to you Telios account us gin TouchID or Face ID without entering the password". @hexadecible @Ellobo16